extensions/blocks/opentable/opentable.php

```diff
@@ -55,18 +32,7 @@ function register_block() {
  * is loaded.
  */
 function set_availability() {
-	if ( is_available() ) {
-		\Jetpack_Gutenberg::set_extension_available( BLOCK_NAME );
-	} else {
-		\Jetpack_Gutenberg::set_extension_unavailable(
-			BLOCK_NAME,
-			'missing_plan',
-			array(
-				'required_feature' => 'opentable',
-				'required_plan'    => 'premium-plan',
-			)
-		);
-	}
+	\Jetpack_Gutenberg::set_availability_for_plan( FEATURE_NAME );
```

Comment:
I wonder if we should use the block name, rather than the feature name, so that we can support premium blocks from other places in WPCOM?

Reply:
We could but the availability array that's created uses the module/feature name as the key, so it's following that pattern for now. We could make some further changes to the availibility code, but that's probably for another PR.